docs/lambda.md

```diff
@@ -39,14 +39,14 @@ For non-node lambdas not included in Cumulus repo, upload .zip to s3 and modify
 For new lambdas, update `<daac>-deploy/lambdas.yml` by adding a new entry.
 E.g.: node.js sample for '../cumulus/cumulus/tasks/sample-lambda' in the Cumulus repo):
 
-    - name: <LambdaName>                                       # eg:  LambdaSample (does not need to conform to dirname)
+    <LambdaName>:                                       # eg:  LambdaSample (does not need to conform to dirname)
       handler: <dir>.<function>                                # eg:  sample-lambda.handler (assuming file has module.exports.handler = <someFunc>)
       timeout: <ms>                                            # eg:  300
       source: '../cumulus/cumulus/tasks/<dir>/dist/<file.js>'  # eg:  '../cumulus/cumulus/tasks/sample-lambda/dist/index.js'
```

Comment:
this path is also incorrect. Also I wonder if this should point to the node_modules folder